### PR TITLE
[W5.11][T16-1]Nur Tristy Erna Binte Mohammed Harlan

### PR DESCRIPTION
**Pull request by tristyxxnana**
added enhancement: changed to find name, ignoring sensitivity of character cases.
user guide updated.
relevant tests updated on JUnit tests and I/O tests

### CODE REVIEW

**Comment by zhicaizack** [comment]
simple clean and easy to understand code, perhaps you could use extract method for refactoring 👍

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -50,8 +50,17 @@ public CommandResult execute() {
...
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
-                matchedPersons.add(person);
+//            if (!Collections.disjoint(wordsInName, keywords)) {
+//                matchedPersons.add(person);
+//            }
+            for (String nameWord : wordsInName) {
+                for (String keyword : keywords) {
+                    if (nameWord.equalsIgnoreCase(keyword)) {
```
Comment (stephlewyh) [inline src/seedu/addressbook/commands/FindCommand.java:58]: This increases the complexity of your function as you have more `for` loops. You may want add a new method that converts `wordsInName` and `keywords` separately, while retaining the `disjoint` function that already checks for the matching between keyword and name for you.  Read up about `Collections.disjoint` here:  https://www.geeksforgeeks.org/java-util-collections-disjoint-method-java-examples/.

**Review by stephlewyh — COMMENTED** [review]
Hi Tristy, 

Good effort!  (1) updated user guide 
(2) added additional test case in JUnit test to test enhancement
(3) provided enhancement code -- could consider other ways of refactoring.

**Comment by tristyxxnana** [comment]
Noted with thanks. Will try to code it more efficiently.

**Pull request closed without merging**